src/nginxconfig/templates/global_sections/security.vue

```diff
@@ -243,4 +261,4 @@ THE SOFTWARE.
             },
         },
     };
-</script>
+</script>
```

Comment:
Not sure what the diff is here, but should probably be reverted?

Reply:
`</script>` was moved down to line 264 because new code was added above, I assume.

Reply:
It moving down would not show as a diff, have you changed the line ending or smth?

Reply:
I haven't really touched anything other than just adding a new data variable in the script block. I even copy pasted the whole thing from the master version to fix the styling issues (quickest way I could do it since eslint wouldn't do it)

Reply:
The diff indicates you've dropped the new line at the end of the file?

Reply:
> The diff indicates you've dropped the new line at the end of the file?

Oh, I completely missed the tooltip. Very sorry about that. I have re-added it.